examples/skills_tuning/instructlab/table_manipulation.py

```
@@ -0,0 +1,639 @@
+# This example demonstrates the reasoning logic required to augment the original table. Note that the answer preserves the original structure but adds a new computed column based on the instruction.
+#
+# For this demo, we’ll use a pre-populated YAML file located at: `seed_data/table_manipulation_qna.yaml`
+#
+# ### 🔁 Convert to JSONL Format
+#
+# Dataflow between blocks and pipelines is handled using Hugging Face Datasets, which are based on Arrow tables. To make the seed data compatible with the generation pipeline, we need to convert the YAML file into a JSONL dataset. This conversion involves extracting the examples from the YAML file and converting them into a dataset.
+
+# %%
+def convert_yaml_to_jsonl(yaml_path):
+    # Load YAML file
+    with open(yaml_path, 'r') as f:
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)